docs/installation/base-url.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Add ability to run Headlamp with a base-url
+linkTitle: Base URL
+---
+
+Normally Headlamp runs at the root of the domain. Hower you can also ask 
+to run it at a base-url like "/headlamp" for example.
+
+- default at the root of the domain: `http://headlamp.example.com/`.
+- base-url `http://example.com/headlamp` 
+
+
+## How to use with a base-url
+
+
+### Dev mode
+
+```bash
+./backend/server -dev -base-url /headlamp
+PUBLIC_URL="/headlamp" make run-frontend
+```
+
+Then go to http://localhost:3000/headlamp/ in your browser.
+
+
+### Static build mode
+
+There is a separate `buildbaseurl` 
+
+```bash
+cd frontend && npm install 
+npm run buildbaseurl
+cd ..
+./backend/server -dev -base-url /headlamp -html-static-dir frontend/build
+```
+
+Then go to http://localhost:4466/headlamp/ in your browser.
+
+
+### Docker mode
+
+Append `--base-url /headlamp` to the docker run command. Note the extra "-".
```

Comment:
Does it really require the two dashes?

Reply:
I found this a bit strange, and I'm not sure why... but yes it does.